### PR TITLE
3093 restrict write permissions after proposal submission

### PR DESCRIPTION
**Pull request by aucuparia**

### CODE REVIEW

File: tools/process-proposal-actions.r [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+	msiGetContInxFromGenQueryOut(*GenQOut, *ContInxNew);
+
+	while(*ContInxOld > 0) {
+		foreach(*row in *GenQOut) {
+			*collName = *row.COLL_NAME;
+
+			# Check if proposal metadata change is requested in datarequests-research group
+			if (*collName like regex "/[^/]+/home/datarequests-research.*") {
+
+                                # Get arguments necessary for processing of metadata change
```
Comment (lwesterhof) [inline tools/process-proposal-actions.r]: Please align

File: uuProposals.r [hunk 2]
```diff
@@ -57,3 +57,198 @@ uuGetProposalsAdditionalFields(*limit, *offset, *attributeName, *result, *status
...
+#
+adminProposalActions() {
+        msiExecCmd("admin-proposalactions.sh", uuClientFullName, "", "", 0,
+                   *out);
+}
+
+
+# \brief Process proposal metadata change request
+#
+# \param[in] proposalColl
```
Comment (lwesterhof) [inline uuProposals.r]: Please add description of parameters

File: uuProposals.r [hunk 3]
```diff
@@ -57,3 +57,206 @@ uuGetProposalsAdditionalFields(*limit, *offset, *attributeName, *result, *status
...
+                foreach(*row in SELECT META_DATA_ATTR_VALUE WHERE
+                                COLL_NAME = *proposalColl AND
+                                DATA_NAME = "proposal.json" AND
+                                META_DATA_ATTR_NAME = *attributeName) {
+                        *currentAttributeValue = *row.META_DATA_ATTR_VALUE;
+                }
+                if (*currentAttributeValue == *newAttributeValue) {
+                        *status = "Success";
+                        *statusInfo = "";
+                        succeed;
```
Comment (lwesterhof) [inline uuProposals.r]: Temporary write ACLs not revoked in this case?

Comment (aucuparia) [reply uuProposals.r]: Good catch.

**Pull request merged**